### ISSUE TITLE
Mask issue : Cannot move feathering points at the bottom , outside of the image

### ISSUE THREAD

**Issue by chhil** [darktable-org/darktable#5195]
Version win64: darktable 3.1.0+1703~g989d68d45

compile options:
  bit depth is 64 bit
  normal build
  SSE2 optimized codepath enabled
  OpenMP support enabled
  OpenCL support enabled
  Lua support enabled, API version 6.0.0
  Colord support disabled
  gPhoto2 support enabled
  GraphicsMagick support enabled
  ImageMagick support disabled
  OpenEXR support enabled

Was trying to retouch an image.
Drew a mask when the image was fit to screen at the bottom of the image and mask crossed the image boundary.
Now I want to pull the feathering in from the bottom and could not access the points. 
Tried zooming in and thought maybe it will allow me to access to those points, but it didn't.
You could argue that there is nothing outside the screen that will impact the retouch, but the feathering points can still be used to form the feathering shape.

![image](https://user-images.githubusercontent.com/1119439/82774659-1507b900-9e63-11ea-8907-2711dba4ff96.png)

**Comment by elstoc** [comment]
Use ctrl+scroll to zoom out further. You should then be able to access and change the points.

You can also use the zoom drop-down on the navigation image on the top-left of the screen. Set it to small, then zoom in as you like.

**Comment by chhil** [comment]
Irrespective of how much one zooms the points are not available.

The points become available when one starts from small and zooms in, this I would consider as a workaround, doesn't feel like a normal workflow.

**Comment by johnny-bit** [comment]
Would you be able in cases like that, move zoom area outside of image just enough to be able to control furthest point outside of image, regardless of zoom setting?

**Comment by chhil** [comment]
@johnny-bit 
What do you mean by move zoom area outside of image?
From what I understand, you zoom a bit and then move the rectangle in the preview top left to get it outside the area, I tried that and could not get to the points.

While moving the rectangle I reproduced https://github.com/darktable-org/darktable/issues/5145 , DT crashed on moving rectangle.

this is darktable 3.1.0+1806~g70e3a7a78 reporting an exception:

-------------------

Error occurred on Sunday, May 31, 2020 at 07:42:19.

```
darktable.exe caused an Access Violation at location 0000000065F45539 in module libtoneequal.dll Reading from location 0000000036884000.

AddrPC           Params
0000000065F45539 000000000635FBF0 0000000064942F28 0000000000000000  libtoneequal.dll!apply_exposure._omp_fn.0  [F:/msys64/home/chill/darktable/src/iop/toneequal.c @ 807]
   805:       for(size_t c = 0; c < ch; ++c)
   806:         out[(i * out_width + j) * ch + c] = (c == 3) ? in[((i + offset_y) * in_width + (j + offset_x)) * ch + 3]  // pass-through alpha mask
>  807:                                                      : in[((i + offset_y) * in_width + (j + offset_x)) * ch + c] *
   808:                                                        correction[(i + offset_y) * in_width + (j + offset_x)];
   809: }
000000000019EAA0 0000000027070EF0 0000000000000000 0000000000000000  libgomp-1.dll!omp_in_final
0000000064945092 0000000026702850 0000000000000000 0000000027070EF0  libwinpthread-1.dll!pthread_create_wrapper
00007FFFDE8DB04A 00007FFFDE9306D0 0000000027070EF0 0000000000000000  msvcrt.dll!_beginthreadex
00007FFFDE8DB11C 0000000000000000 0000000000000000 0000000000000000  msvcrt.dll!_endthreadex
00007FFFE05B7BD4 0000000000000000 0000000000000000 0000000000000000  KERNEL32.DLL!BaseThreadInitThunk
00007FFFE07ECE51 0000000000000000 0000000000000000 0000000000000000  ntdll.dll!RtlUserThreadStart
```

**Comment by johnny-bit** [comment]
This issue is earlier, however #6307 provides interesting method of getting access to mask points... Would you mind if we closed this in favour of #6307?

**Comment by johnny-bit** [comment]
Let's continue in https://github.com/darktable-org/darktable/issues/6307 OK?
